src/doc/guide.md

```diff
@@ -4231,7 +4231,7 @@ arguments, really powerful things are possible.
 
 Let's make a closure:
 
-```{rust}
+```{rust,ignore}
```

Comment:
@steveklabnik you need a stage1 compiler (built with this PR) to see the compiler errors in the doctests. Let me update the snippets and then you can update the text around. (That should be faster since I have a stage1 compiler at hand).